File: data/CodeLlama/readable_format_HPEC/fortran/file134.f90
program DRB090_static_local_orig_yes
    use omp_lib
    implicit none

    integer :: i, len
    integer, dimension(:), allocatable :: a, b
    integer, save :: tmp
    integer :: tmp2

    len = 100
    allocate (a(len))
    allocate (b(len))

    do i = 1, len
        a(i) = i
        b(i) = i
    end do

    !$omp parallel
        !$omp do
        do i = 1, len
            tmp = a(i)+i
            a(i) = tmp
        end do
        !$omp end do
    !$omp end parallel

    !$omp parallel
        !$omp do
        do i = 1, len
            tmp2 = b(i)+i
            b(i) = tmp2
        end do
        !$omp end do
    !$omp end parallel

    print 100, a(50), b(50)
    100 format (i3,3x,i3)

    deallocate(a,b)

end program

File: data/gpt-4o/readable_format_HPEC/cpp/file134.cpp
#include <stdio.h>
#include <omp.h>
int a[100], b[100];
int main()
{
 int i;
 int tmp, tmp2;
 for (i=0;i<100;i++)
 {
 a[i]=i+1;
 b[i]=i+1;
 }
#pragma omp parallel private(tmp)
 {
#pragma omp for
 for (i=0;i<100;i++)
 {
 tmp=a[i]+(i+1);
 a[i]=tmp;
 }
 }
#pragma omp parallel private(tmp2)
 {
#pragma omp for
 for (i=0;i<100;i++)
 {
 tmp2=b[i]+(i+1);
 b[i]=tmp2;
 }
 }
 printf ("%d   %d\n", a[49], b[49]);
 return 0;
}
